### ISSUE TITLE
Error when refreshing subscriptions

### ISSUE THREAD

**Issue by antoine1619** [SimonDaNinja/youtube_rss#6]
I'm on MacOS 10.15.7
Upon refreshing subscriptions, very briefly, I get an error message:
I can't see the whole thing, but here is what it partially says:
```
refreshing subscriptions ... Process Process-2:
Traceback (most recent call last):
File "/usr/local/Cell
hona3.9/3.9.5/Frameworks/Python.framework/Versions/3.9/lib/python3.9/multiprocessing/process.py", line 315. in bootstrap self.run( )
Python.framework/Versions/3.9/lib/python3.9/multiprocessing/process.py", line 108, in run Fite /usr/Local/Cellar/pythonas. 9/3.9.5/Fram
self._target(*self._args, **self._wargs)
77 in refreshSubsriptionsByChannelss File "/Users/#####/youtube_rss/youtube_rss.py"
if USE THUMBNAILS:
NameError: name 'USE THUMBNAILS' is not defined

```

**Comment by SimonDaNinja** [comment]
Oh, that's strange, and definitely not good... probably a symptom of multiprocessing being handled differently on Mac compared to Linux. I think I have an idea what might fix it, but it'll have to wait until am not at work. Thanks for letting me know!

**Comment by SimonDaNinja** [comment]
@toylatte Do you get this problem consistently? (Would make it easier to confirm if any changes I make solve the problem)

**Comment by antoine1619** [comment]
Yes, this happens everytime I refresh.

**Comment by SimonDaNinja** [comment]
@toylatte I think the last commit, 16036f4e6e4e231a84a51be742d5e91bb95f359c, might fix this. Could you pull and see if you still have the same issue?

**Comment by antoine1619** [comment]
It works now, thanks

**Comment by antoine1619** [comment]
Now everytime I finish a video, I get an error saying that `database` is not defined.
`Traceback (most recent call last):
  File "/Users/#####/youtube_rss/youtube_rss.py", line 1279, in <module>
    doMainMenu(runtimeConstants)
  File "/Users//#####//youtube_rss/youtube_rss.py", line 1204, in doMainMenu
    doMethodMenu("What do you want to do?", menuOptions)
  File "/Users//#####//youtube_rss/youtube_rss.py", line 1218, in doMethodMenu
    result = methodMenuDecision.executeDecision()
  File "/Users//#####//youtube_rss/youtube_rss.py", line 323, in executeDecision
    return self.function(*self.args, **self.kwargs)
  File "/Users//#####//youtube_rss/youtube_rss.py", line 1068, in doInteractiveBrowseSubscriptions
    doMethodMenu("Which channel do you want to watch a video from?", menuOptions,
  File "/Users//#####//youtube_rss/youtube_rss.py", line 1218, in doMethodMenu
    result = methodMenuDecision.executeDecision()
  File "/Users//#####//youtube_rss/youtube_rss.py", line 323, in executeDecision
    return self.function(*self.args, **self.kwargs)
  File "/Users//#####//youtube_rss/youtube_rss.py", line 1094, in doSelectVideoFromSubscription
    doMethodMenu("Which video do you want to watch?", menuOptions, adHocKeys=adHocKeys)
  File "/Users//#####//youtube_rss/youtube_rss.py", line 1218, in doMethodMenu
    result = methodMenuDecision.executeDecision()
  File "/Users//#####//youtube_rss/youtube_rss.py", line 323, in executeDecision
    return self.function(*self.args, **self.kwargs)
  File "/Users//#####//youtube_rss/youtube_rss.py", line 1102, in doPlayVideoFromSubscription
    outputDatabaseToFile(database, DATABASE_PATH)
NameError: name 'database' is not defined`

**Comment by SimonDaNinja** [comment]
Ah, it's one of many bugs that creeped up after my big threading-refactoring. Since I changed code in so many places, there are some loose ends here and there that I have missed. Thanks a lot for helping me find these issues!

It should be solved in 88cf4bc6408a7927b05976a8100bb1522c224bae, so if you pull, it shouldn't be an issue anymore.

**Comment by antoine1619** [comment]
It works now, thanks
